### DISCUSSION TITLE
Add Support for few A11 roms

### DISCUSSION THREAD

**Discussion by Ranger-XD** [ROM-builders/temporary#8882] [Ideas]
Sir , As u know , A12 is laggy for many devices community so can u plg add some a11 roms like komodo os , msm - extended & aosp - extended.
& your choice which u wana add.
Thanks For cirrus ci 😊

**Comment by Apon77** [comment]
> Sir , As u know , A12 is laggy for many devices community so can u plg add some a11 roms like komodo os , msm - extended & aosp - extended. & your choice which u wana add. Thanks For cirrus ci 😊

1. if a11 source is updated regularly, or rom is too much popular, only then it can be added in supported list. and obviously you have to do proper pr!! for adding support of roms or branches

**Reply by Ranger-XD** [reply]
Oh thanks sir , 
Then u can add LTS roms
I will give some rom names soon🤩

**Comment by Nishant9947** [comment]
Sir , what about to add evolution x A11 Lts rom ?
Must consider about it

**Reply by Apon77** [reply]
> Sir , what about to add evolution x A11 Lts rom ? Must consider about it

whats special in it?

**Reply by Nishant9947** [reply]
LTS SIR LONG TERM SUPPORT ,
Who want to stay on a11 because a12 has not much performance than a11 cause of heavy animation. So he can get updates regularly with lts roms like Evo x or Nusantara os.

Don't mind sir , I am suggesting only😊
If you think it is good then you can tell me i will open  PR or you can do it

**Reply by Apon77** [reply]
> LTS SIR LONG TERM SUPPORT , Who want to stay on a11 because a12 has not much performance than a11 cause of heavy animation. So he can get updates regularly with lts roms like Evo x or Nusantara os.
> 
> Don't mind sir , I am suggesting only😊 If you think it is good then you can tell me i will open PR or you can do it

if rom is not updated regularly in source, we should not support it i think
